components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/session/extender/response/SessionExtenderErrorResponseTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authentication.framework.session.extender.response;
+
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertNotNull;
+
+/**
+ * Unit test cases for SessionExtenderResponse.
+ */
+public class SessionExtenderErrorResponseTest extends PowerMockTestCase {
+
+    @Test
+    public void buildTestErrorResponse() {
+
+        SessionExtenderErrorResponse.SessionExtenderErrorResponseBuilder builder =
+                new SessionExtenderErrorResponse.SessionExtenderErrorResponseBuilder();
+        builder.setErrorCode("Sample error code");
+        builder.setErrorMessage("Sample error message");
+        builder.setErrorDescription("Sample description.");
+        builder.setTraceId("1234-5678");
+        SessionExtenderErrorResponse response = builder.build();
+        assertNotNull(response.getResponse(), "Error creating failure response.");
```

Comment:
Shouldn't we assert the contents of the response?

Reply:
Fixed in 994a28a9538ac6c2d019880010c13ce063c53a1c